src/styles/pages/style.scss

```diff
@@ -0,0 +1,844 @@
+body {
+  font-size: 16px;
+  line-height: 1.5;
+  color: #F5F8FB;
+  font-family: 'Bakbak One', Arial, sans-serif;
+  background: #0D0D15;
+}
+
+/* Bakbak One for headings */
+.bakbak-one-regular {
+  font-weight: 400;
+  font-style: normal;
+}
+
+/* Poppins for body text */
+.poppins-medium {
+  font-family: 'Poppins', Arial, sans-serif;
+  font-weight: 500;
+  font-style: normal;
+}
+
+/* Header */
+.header {
+  width: 1728px;
+  height: 112px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 20px;
+}
+
+.logo {
+  display: flex;
+  align-items: center;
+}
+
+.logo img {
+  width: 188px;
+  height: 42px;
+}
+
+.logo__image {
+  height: 40px;
+}
+
+/* Navigation */
+
+.main-nav {
+  width: 401px;
+  height: 40px;
+  display: flex;
+  align-items: center;
+}
+
+.main-nav__list {
+  width: 89px;
+  /* Fixed missing unit */
+  height: 40px;
+  gap: 8px;
+  padding-top: 8px;
+  padding-right: 20px;
+  padding-bottom: 8px;
+  padding-left: 20px;
+  border-bottom-width: 2px;
+  list-style: none;
+  display: flex;
+  margin: 0;
+  padding: 0;
+}
+
+.main-nav__link {
+  font-family: 'Poppins', Arial, sans-serif;
+  font-weight: 600;
+  font-size: 16px;
+  color: #F5F8FB;
+  text-decoration: none;
+  padding: 10px 15px;
+  width: 49px;
+  height: 24px;
+  line-height: 24px;
+  letter-spacing: 0px;
+  text-align: center;
+  vertical-align: middle;
+  color: #ffffff;
+  font-weight: 500;
+}
+
+.main-nav__link:hover {
+  text-decoration: underline;
+}
+
+.main-nav__list-item {
+  margin-right: 20px;
+}
+
+/* .main-nav__list-item {
+  font-family: 'Poppins', sans-serif;
+  font-weight: 600;
+  font-size: 16px;
+  line-height: 1.5;
+  color: #F5F8FB;
+  margin-right: 20px;
+} DO NOT DUPLICATE STYLES. IF YOU DO NOT HAVE TEXT CONTENT INSIDE THE TAG/SELECTOR */
+
+
+.search {
+  padding: 5px;
+  margin-right: 20px;
+  border: 1px solid #333;
+  border-radius: 4px;
+  background-color: #2c2c2c;
+  color: #ffffff;
+}
+
+
+/* Hero Banner */
+.banner {
+  width: 866.500732421875px;
+  height: 362px;
+  background: radial-gradient(44.91% 42.19% at 22% -4%, rgba(2, 101, 220, 0.5), rgba(2, 101, 220, 0) 100%), rgb(13, 13, 21);
+}
+
+.banner__heading {
+  font-size: 48px;
+  margin-bottom: 20px;
+  line-height: 1.2;
+}
+
+.banner__highlight {
+  color: #0265DC;
+}
+
+.banner__text {
+  font-family: 'Poppins', Arial, sans-serif;
+  font-size: 18px;
+  margin-bottom: 30px;
+  line-height: 1.5;
+}
+
+.banner__stats {
+  display: block;
+  width: 100%;
+  max-width: 800px;
+}
+
+.banner__stat-label {
+  font-family: 'Poppins', Arial, sans-serif;
+  width: 95px;
+  height: 24px;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  letter-spacing: 0px;
+  vertical-align: bottom;
+}
+
+.banner__stat-value {
+  box-shadow: 0px 4px 13px 0px rgba(255, 255, 255, 0.4);
+  font-size: 52px;
+  line-height: 62px;
+  letter-spacing: -0.5px;
+  vertical-align: bottom;
+}
+
+.banner__image {
+  width: 533.499267578125px;
```

Comment:
Round pixel values: Instead of using overly precise values like 533.499267578125px, round to the nearest tenth when replicating from Figma, e.g. 533.5px.